File: SFA-API/Domain/StoreItemBatch.cs
namespace Domain
{
    public class StoreItemBatch
    {
        public int Id { get; set; }
        public int StoreId { get; set; }
        public int ItemBatchId { get; set; }
        public int ItemCount { get; set; }
        public virtual Store Store { get; set; }
        public virtual ItemBatch ItemBatch { get; set; }
    }
}

File: SFA-Client/src/app/_models/storeItemBatch.ts
export class StoreItemBatch {
  id: number;
  storeId: number;
  storeName: string;
  itemBatchId: number;
  itemBatchName: string;
  itemCount: number;

  constructor(storeItemBatch) {
    this.id = storeItemBatch.id;
    this.storeId = storeItemBatch.storeId;
    this.storeName = storeItemBatch.storeName;
    this.itemBatchId = storeItemBatch.itemBatchId;
    this.itemBatchName = storeItemBatch.itemBatchName;
    this.itemCount = storeItemBatch.itemCount;
  }
}
